### ISSUE TITLE
User Features - Owners Information Should Come Up With Dog Picture When Viewing Pack Members

### ISSUE THREAD

**Issue by gudell** [gudell/bwardp#169]
I thought we talked about having the spinning tile that would reveal the owner?  Not sure though

**Comment by glennclarke** [comment]
Yes, I though that we had implemented this? If its a bug, lets fix it. If it was never implemented lets hold it for later.

**Comment by zakwhit** [comment]
Ran out of time. The flipping image is only activated in the user's profile page.

**Comment by gudell** [comment]
Let's make sure this gets captured in the next update
